src/background.js

```diff
@@ -289,6 +289,23 @@ function getRootDomain(url) {
 
 }
 
+function getFrameAncestorsURLAndCheckIfFacebook(frameAncestorsArray) {
```

Comment:
And this could be `topFrameUrlIsFacebookApps` ?

Reply:
Revised!